packages/hooks/src/useCFDDemoAccounts.ts

```diff
@@ -9,12 +9,7 @@ const useCFDDemoAccounts = () => {
     const { getRate } = useExchangeRate();
     const cfd_accounts = useCFDAllAccounts();
 
-    const cfd_demo_accounts = cfd_accounts
-        .filter(account => account.account_type === 'demo')
-        .map(account => ({
-            ...account,
-            balance: account.balance ? account.balance / getRate(account.currency || '') : account.balance,
-        }));
+    const cfd_demo_accounts = cfd_accounts.filter(account => account.account_type === 'demo');
```

Comment:
As you have reverted the changes so there is no need to import `useExchangeRate` hook. So remove the line#2 and line#9.

Reply:
It is being called in line 13 😆